### DISCUSSION TITLE
Start Menu search box doesn't work

### DISCUSSION THREAD

**Discussion by topwizprogramming** [valinet/ExplorerPatcher#2909] [General]
ExplorerPatcher Version 22621.3007.63.3
Windows 11 Version 22631.3155

I click on the Start or Search button and start typing the name of a program. The search window comes up but never finds anything.

**Comment by pyrates999** [comment]
Please include what settings you changed in explorer patcher properties application.  These should be screen shots.

Please include any other software you have installed that modifies the UI of windows.

**Comment by topwizprogramming** [comment]
I have Winaero Tweaker installed but haven't done anything in a long time. My ExplorerPatcher uninstall was messed up so I had to reinstall and then uninstall. Now the search box works correctly.

**Comment by pyrates999** [comment]
What settings did you change in winaero tweaker?

**Comment by topwizprogramming** [comment]
No idea, it was too long ago.

**Comment by pyrates999** [comment]
try this: https://www.majorgeeks.com/content/page/how_to_delete_and_rebuild_the_windows_10_search_index.html
